doc/conflicts.md

```diff
@@ -174,6 +198,16 @@ It is also possible to return a `CDTDocumentRevision` from
     return rev;
 }
 ```
+```swift
+class CDTMergeResolver : NSObject, CDTConflictResolver {
+    func resolve(docId: String!, conflicts: [AnyObject]!) -> CDTDocumentRevision! {
+        let rev = conflicts.first!
+        rev.body // ...update body, perhaps with data from the other conflicts
+        rev.attachments // ...you can also create/update/delete attachments
```

Comment:
Missing `=` on the two above lines.

I'd also prefer to see block comments as with the objc.